modulus/launch/logging/launch.py

```diff
@@ -324,6 +323,14 @@ def _log_backends(
             metric_dict[step[0]] = step[1]
             wandb.log(metric_dict)
 
+    def log_mlflow_figure(self, figure, artifact_file: str = "artifact"):
```

Comment:
lets change this to `log_figure`... launch logger should not have specific back end functions, rather general ones that then have the logic to figure out how to handle specific backends.

Reply:
Great!